templates/home.html

```diff
@@ -38,29 +103,68 @@ <h4 class="white-color header-padding">Welcome, {% firstof user.get_full_name us
                     background-color: black;
                 }
             </style>
+            <div class="header-position">
+                <h2 class="white-color header-text">Investment Platform for Creatives</h2>
+                <h4 class="white-color header-padding">Start your own record label and build your brand in music</h4>
+            </div>
             <div class="footer-position">
-                <img class="header-logo" src="{% static "img/perdiem-home-stroke.svg" %}">
-                <h1 class="white-color header-text">PerDiem</h1>
-                <h4 class="white-color header-padding header-text">A new way to invest in music</h4>
-                <h5 class="white-color header-text">
-                    Earn money by discovering and investing in artists
-                </h5>
-                <br />
-                <h5>
-                    <a href={% url 'home' %}#about><i class="fa fa-chevron-down fa-1x color-grey" aria-hidden="true"></i></a>
-                </h5>
+                <a href={% url 'home' %}#about>
```

Comment:
Looks like this was an issue before, but I missed it. We're missing quotes around the URL here, which will make this HTML not work on some browsers.